packages/launch/src/hooks/use-title.ts

```diff
@@ -28,8 +38,28 @@ export function useTitle() {
 		debouncedSaveSiteTitle( siteId, launchSiteTitle );
 	}, [ launchSiteTitle, debouncedSaveSiteTitle, siteId, noLaunchSiteTitle ] );
 
+	const DEFAULT_SITE_NAME = __( 'Site Title', __i18n_text_domain__ );
+	let isDefaultTitle = typeof title !== 'undefined' && title === DEFAULT_SITE_NAME;
```

Comment:
```suggestion
	let isDefaultTitle = title === DEFAULT_SITE_NAME;
```

I think that the `typeof title !== 'undefined'` check shouldn't be added here.
Checking for `title === DEFAULT_SITE_NAME` should be enough — plus this check would potentially break other parts of the code (see this [comment](https://github.com/Automattic/wp-calypso/pull/50311/files#r580458487))